packages/nimble-components/src/rich-text-viewer/tests/rich-text-viewer.stories.ts

```diff
@@ -27,8 +30,26 @@ const metadata: Meta<RichTextViewerArgs> = {
         componentName: 'rich text viewer',
         statusLink: 'https://github.com/ni/nimble/issues/1288'
     })}
-        <${richTextViewerTag}></${richTextViewerTag}>
-    `)
+    <${richTextViewerTag}
+        :markdownValue="${x => x.markdownValue}"
+        ?fit-to-content="${x => x.fitToContent}"
```

Comment:
When I played with `fit-to-content` in Storybook I didn't see the behavior I was expecting. If I set it to `false` and then add a lot of content, it seems like the text viewer element keeps growing in height. Am I misunderstanding the expected behavior or is it misbehaving?

Reply:
The reason I added `fit-to-content` is to grow the height of the component to fit the content even when there is a height set to the component or parent component by the consumer. But as mentioned by Milan in [this comment](https://github.com/ni/nimble/pull/1335#discussion_r1260418995), it is better not to have this attribute for the viewer component. So I have removed the attribute from all files and updated the spec as well.